changelog/@unreleased/pr-593.v2.yml

```diff
@@ -0,0 +1,18 @@
+type: improvement
+improvement:
+  description: |-
+    Reverse order of computed resolvers list
```

Comment:
Was this generated by the app? This content is great for the PR template, but is not the right format/content for the changelog, which is primarily meant to be consumed as release notes by the user.

The release note should focus more on the observable change in behavior rather than on the implementation details -- it should basically say that, if resolvers are specified for defaults tasks, those are now used first before the built-in/default resolver. May also be worth adding that the previous behavior can be maintained after this change by adding the built-in resolver as the first entry in the custom resolver list if specified in config.

Reply:
Yup, it was autogenerated.  I've updated it to be more succinct.